content/intermediate/201_resource_management/cleaning.md

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
Please rename the _file_ from `cleaning.md` to `cleanup.md` to match with other modules.